### ISSUE TITLE
Possibility to remove link from a breadcrumb

### ISSUE THREAD

**Issue by CNicolas** [udayvunnam/xng-breadcrumb#31]
**Is your feature request related to a problem? Please describe.**
If my breadcrumbs look like this : `Pages / ShadowGroup / Page1`, I want to remove the routerLink from the `ShadowGroup`. It can occurs when you  need to explain to the user where he is, but without giving a route that has no meaning

**Describe the solution you'd like**
At least an option like `data: { breadcrumb: {unlink: true, label: 'ShadowGroup'}`. The default style should be like a hint-text.
The best would be to pass the full template to the breadcrcumb and add a `<a>` or not based on the `data.info`.

**Describe alternatives you've considered**
I thought about reorganize routes to have only `ShadowGroup / Page1`, but you lose the precision of the `Pages` parent.
I can also redirectTo, but it's a real mess for the user to understand what's happening.

**Comment by udayvunnam** [comment]
Hi @CNicolas, this would have been possible by, passing additional context to breadcrumb and stopping the click events for route patterns that don't need it. 

- define 'info' associated with the breadcrumb in route config. 'info' has type <any>. you can pass string or object or boolean as you need.
- 'info' is available in the context of _\*xngBreadcrumbItem_.

```javascript
  {
    path: '',
    pathMatch: 'full',
    component: HomeComponent,
    data: {
      breadcrumb: {
        label: 'app home',
        info: { skipLink: true }
      }
    }
  }
```

```html
<xng-breadcrumb>
  <ng-container *xngBreadcrumbItem="let breadcrumb; let info = info>
    <span [ngClass]="{'skip-link': info && info.skipLink}">{{ breadcrumb }}</span>
  </ng-container>
</xng-breadcrumb>
```

```CSS
.skip-link {
   pointer-events: none;
   cursor: default;
}
```

Since the customization control is given only to breadcrumb text but not the <a> or <label>, the above implementation will not work.

Two possible solutions: 
- Give the customization control at <a> or <label> level(more work for the user and I am not inclined towards this solution.)
**- provide an extra option like `unlink: true` in the breadcrumb. I will implement this and release a new version.**

Thanks for raising this feature request. I will keep you posted!

**Comment by dhallatmaine** [comment]
I also have a need for this functionality now.

**Comment by udayvunnam** [comment]
In progress - @dhallatmaine. Will release in a week

**Comment by udayvunnam** [comment]
@CNicolas @dhallatmaine This feature is available with xng-breadcrumb@5.1.0 onwards.

please check https://github.com/udayvunnam/xng-breadcrumb#dibsable-breadcrumb-navigation-for-certain-routes
